delft3dworker/views.py

```diff
@@ -262,7 +332,22 @@ class UserViewSet(viewsets.ModelViewSet):
     serializer_class = UserSerializer
 
     def get_queryset(self):
-        return User.objects.all()
+        user = get_object_or_404(User, id=self.request.user.id)
+        wanted = [group.id for group in user.groups.all()]
```

Comment:
This will also return the users in group access:world... which is everyone anyways.